Gemfile

```diff
@@ -4,6 +4,3 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in annealing.gemspec
 gemspec
-
-gem 'minitest', '~> 5.0'
```

Comment:
I'll rather to keep the dev dependencies in gem file since its the default from bundler.